<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>534</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>224</width>
    <height>278</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Space Invaders</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>448</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_ROM"/>
    <addaction name="actionClose_ROM"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="action_Re_Start_Game"/>
    <addaction name="actionPause_Game"/>
    <addaction name="actionInsert_Coin"/>
    <addaction name="actionP1_Start"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionRun_Video_Test"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGame"/>
   <addaction name="menuDebug"/>
  </widget>
  <action name="actionLoad_ROM">
   <property name="text">
    <string>Load ROM</string>
   </property>
  </action>
  <action name="action_Re_Start_Game">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>(Re)Start Game (R)</string>
   </property>
  </action>
  <action name="actionPause_Game">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pause Game (P)</string>
   </property>
  </action>
  <action name="actionInsert_Coin">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Insert Coin (C)</string>
   </property>
  </action>
  <action name="actionRun_Video_Test">
   <property name="text">
    <string>Run Video Test</string>
   </property>
  </action>
  <action name="actionClose_ROM">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close ROM</string>
   </property>
  </action>
  <action name="actionP1_Start">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>P1 Start (Enter)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
